app/components/navbar.tsx

```diff
@@ -235,6 +239,78 @@ function MobileMenu() {
   )
 }
 
+// Timing durations used to control the speed of the team ring in the profile button.
+// Time is seconds per full rotation
+const durations = {
+  initial: 20,
+  hover: 10,
+  focus: 4,
+  active: 0.25,
+}
```

Comment:
You can play a bit with these to get speeds that look good to you. The lower the value, the faster it rotates.